src/ui/components/Header/Header.js

```diff
@@ -78,6 +78,7 @@ function HeaderTitle({ recordingId, editingTitle, setEditingTitle }) {
   }
 
   const { recordingTitle, title } = data.recordings?.[0] || {};
+  console.log(data, recordingTitle, title);
```

Comment:
did you mean to leave this console.log in here?
```suggestion
```

Reply:
haha - sloppy :)